nodejs/eks/nodegroup.ts

```diff
@@ -246,6 +257,10 @@ export function createNodeGroup(name: string, args: NodeGroupOptions, parent: pu
         }
     }
 
+    if (args.cloudFormationTags && args.autoScalingGroupTags) {
```

Comment:
I’m fine with this as is - but fwiw - this is not what I was suggesting in my feedback. I can imagine there are technically scenarios where the user wants to provide both of these (for different tags) - but we could just document that it is not correct for the user to provide the same tag in both of these. It can then be up to the user to ensure they put the desired tag in one or the other but not both of these.

Reply:
I've updated the PR to allow both tags to be set, now with proper docs on their usage.